### PR TITLE
Improve test coverage

### PR DESCRIPTION
**Pull request by blackwinter**

### CODE REVIEW

File: src/test/java/hbz/limetrans/filter/LibraryMetadataFilterTest.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    }
+
+    @Test
+    public void testAnyEquals() throws IOException {
+        testEqualsReference("any-equals", "Print version:");
+    }
+
+    @Ignore("org.culturegraph.mf.metamorph.functions.Regexp can't handle null values")
+    @Test
+    public void testAnyRegexp() throws IOException {
```
Comment (blackwinter) [inline src/test/java/hbz/limetrans/filter/LibraryMetadataFilterTest.java]: MAJOR Fix or remove this skipped unit test [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1607)

**Comment by blackwinter** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: src/test/java/hbz/limetrans/filter/LibraryMetadataFilterTest.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    }
+
+    @Test
+    public void testAnyEquals() throws IOException {
+        testEqualsReference("any-equals", "Print version:");
+    }
+
+    @Ignore // org.culturegraph.mf.metamorph.functions.Regexp can't handle null values
+    @Test
+    public void testAnyRegexp() throws IOException {
```
Comment (blackwinter) [inline src/test/java/hbz/limetrans/filter/LibraryMetadataFilterTest.java]: MAJOR Fix or remove this skipped unit test [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1607)

File: src/test/java/hbz/limetrans/filter/LibraryMetadataFilterTest.java (hunk 1 above)
Comment (blackwinter) [inline src/test/java/hbz/limetrans/filter/LibraryMetadataFilterTest.java:59]: MAJOR Fix or remove this skipped unit test [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1607)

**Comment by blackwinter** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [LibraryMetadataTransformation.java#L18](https://github.com/hbz/limetrans/blob/9d84ae85043df9e106c4acf3afde76cadf707b3b/src/main/java/hbz/limetrans/LibraryMetadataTransformation.java#L18): Remove this unused import 'java.util.Map'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Comment by blackwinter** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

**Review by philboeselager — APPROVED** [review]

**Pull request merged**